### DISCUSSION TITLE
How to minimise style & layout impact on main thread work

### DISCUSSION THREAD

**Discussion by alex-r89** [stitchesjs/stitches#950] [Help]
Hi there,

I have recently migrated my site over to stitches (from styled components) and really like some of the features.

One issue I am facing though is that my Google Lighthouse performance score is let down the style & layout impact on the main thread work:

<img width="910" alt="Screenshot 2022-02-13 at 11 02 58 pm" src="https://user-images.githubusercontent.com/7352884/153779257-42452501-4e66-48e2-ae12-24a66915b638.png">

I'm assuming this is some how caused by stitches and was wondering if there are any tips on improving this?

I am using this with NextJS essentially copying the way its done in the NextJS with stiches example. This is using `@stitches/react` version `1.2.6`.

Thanks

**Comment by hadihallak** [comment]
That's very interesting and i'm very keen on knowing the reasons behind this. would you be able to provide a minimal reproduction of the issue? like a codesandbox with Stitches and with SC so that we could easily compare and debug what's behind this
